core/src/main/java/com/klaytn/caver/contract/ContractMethod.java

```diff
@@ -248,22 +247,233 @@ public ContractMethod(Caver caver, String type, String name, List<ContractIOType
      * @throws InvocationTargetException
      */
     public TransactionReceipt.TransactionReceiptData send(List<Object> arguments, SendOptions options, TransactionReceiptProcessor processor) throws IOException, TransactionException, ClassNotFoundException, NoSuchMethodException, InstantiationException, IllegalAccessException, InvocationTargetException {
+        SendOptions determinedOption = makeSendOption(options);
+        AbstractTransaction transaction = sign(arguments, determinedOption);
+
+        if(determinedOption.isFeeDelegation() && determinedOption.getFeePayer() != null) {
```

Comment:
What will happen if `determinedOption.isFeeDelegation()` is true, but `determinedOption.getFeePayer()` is null?

Reply:
It will catch and throw exception in KeyringContainer@getKeyring. 
The error message is "Invalid address. To get keyring from wallet, you need to pass a valid address string as a parameter."

It would be better to check whether fee payer address is null or not in this code. I'll fix it.